delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/SparqlLink.scala

```diff
@@ -71,21 +71,21 @@ object SparqlLink {
       for {
         link         <- SparqlExternalLink(bindings)
         project      <-
-          bindings.get(nxv.project.prefix).map(_.value).flatMap(Iri.absolute(_).toOption).flatMap(projectRefFromId)
```

Comment:
This was renamed to follow a naming change in Jena which now follows a different RFC for IRIs.